### DISCUSSION TITLE
Unable to install Spatie/ray

### DISCUSSION THREAD

**Discussion by samdgea** [spatie/ray#693] [Q&A]
**Describe the bug**
Unable to install the spatie/laravel-ray to the existing laravel project

**Versions**
You can use `composer show` to get the version numbers of:
- spatie/ray package version: `2.0.1`
- spatie/laravel-ray package version (if applicable):  `^1.29`
- PHP version: `7.4.28`
- Laravel version (if applicable): `8.9.0`

**To Reproduce**
1. Open existing project
2. run `composer require spatie/laravel-ray --dev`
3. See error

**Screenshots**
![image](https://user-images.githubusercontent.com/43635222/167540724-a9ea21be-afcf-40a1-9a4c-9bb6d6687e8d.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Version 22.04

**Comment by freekmurze** [comment]
Seems like you are trying to install the package in an unsupported version of Laravel. Our test indicate that the package is not in error.
